Sources/Paywalls/Components/Common/StackableComponent.swift

```diff
@@ -20,7 +20,6 @@ public extension PaywallComponent {
 
     protocol StackableComponent {
 
-        var components: [PaywallComponent] { get }
```

Comment:
A `StackableComponent` without child `components` seems odd, but I guess the entire `StackableComponent` is gonna go, right?

Reply:
Yeah yeah yeah, is going to go in next PR 😇